README.md

```diff
@@ -3,37 +3,52 @@ Automation Tools
 
-Suggested deployment is to use cron to run a shell script that runs the automate transfer tool. Example shell script:
+### Configuration - Overview
 
-    #!/bin/bash
-    cd /usr/lib/archivematica/automation-tools/transfers/
-    /usr/share/python/automation-tools/bin/python transfer.py --user <add user>  --api-key <add api key> --transfer-source <add transfer source location uuid> --depth 2
+Suggested deployment is to use cron to run a shell script that runs the automate transfer tool. Example shell script (for example in `/etc/archivematica/automation-tools/transfer-script.sh`):
 
-This script is run through a crontab entry. Example:
+```
+#!/bin/bash
+cd /usr/lib/archivematica/automation-tools/transfers/
+/usr/share/python/automation-tools/bin/python transfer.py --user <user>  --api-key <apikey> --transfer-source <transfer_source_uuid> --config-file <config_file>
+```
 
-    */5 * * * * /etc/archivematica/automation-tools/transfer-script.sh
+The script can be run from a shell window like:
 
-The cron entry executes the `transfer-script.sh` script. This should be run as the same user as Archivematica is run as (likely the `archivematica` user.)
+```
+user@host:/etc/archivematica/automation-tools$ sudo -u archivematica ./transfer-script.sh
+```
 
-When running, automate transfers stores its working state in transfers.db, a sqlite database.  It contains a record of all the transfers that have been processed.  In a testing environment, deleting this file will cause the tools to re-process any and all folders found in the Transfer Source Location. 
+It is suggested to run the script through a crontab entry for user archivematica (to avoid the need to repeatedly invoke it manually):
 
-Configuration
--------------
+```
+*/5 * * * * /etc/archivematica/automation-tools/transfer-script.sh
+```
 
-This script can be modified to adjust how automated transfers work.  The full set of parameters that can be changed are:
+When running, automated transfers stores its working state in a sqlite database.  It contains a record of all the transfers that have been processed.  In a testing environment, deleting this file will cause the tools to re-process any and all folders found in the Transfer Source Location.
+
+### Configuration - Parameters
+
+The `transfers.py` script can be modified to adjust how automated transfers work.  The full set of parameters that can be changed are:
 
 * `-u USERNAME, --user USERNAME` [REQUIRED]: Username of the dashboard user to authenticate as.
 * `-k KEY, --api-key KEY` [REQUIRED]: API key of the dashboard user.
 * `-t UUID, --transfer-source UUID`: [REQUIRED] Transfer Source Location UUID to fetch transfers from. Check the next section for more details on this field.
+* `-c FILE, --config-file FILE`: [OPTIONAL] config file containing locations and file names for log/database/PID files. If not specified default values are used.
```

Comment:
I had only added [REQUIRED] to required options - I didn't mark optional things. Remove [OPTIONAL] or add it to the rest of the options. Also, what's the default behaviour?